experiment/quick-sort/simulation/css/quicksort-styles.css

```diff
@@ -80,3 +105,7 @@
     text-align: center;
     padding-top: 2%;
 }
+
+.legend { list-style: none; }
+.legend li { padding-bottom : 1.5vw; width: 20vw; }
```

Comment:
Try to keep the formatting same as above CSS blocks, don't write it in a single lin